revdep/reverse-imports-urls

```diff
@@ -0,0 +1,48 @@
+http://github.com/soodoku/abbyyR
```

Comment:
This file should be `.gitignore`'d imo, compare `revdepcheck` which only versions `failures.md`, `problems.md` and `README.md`.

Reply:
that makes sense, I did have in mind a way for users to report their repos to us if they're not linked on CRAN. but that should be in a separate file since this one's automatic and the other manual. maybe reverse-import-urls-(public vs private) ?

Reply:
Maybe something like `extra-check-urls` could be developed if and when the need arises.

Reply:
i'll leave it in for now for easier reviewing but will remove+ignore before merging this